commands/acl-setuser.md

```diff
@@ -40,11 +53,14 @@ The following documentation is a reference manual about the capabilities of this
 
 ## List of rules
 
+Redis ACL rules are split into two categories, rules that define command permissions, "Command rules", and rules that define user state, "User management rules".
```

Comment:
```suggestion
Redis ACL rules are split into two categories: rules that define command permissions, "Command rules", and rules that define user state, "User management rules".
```

and possibly, use *emphasis* instead of "quotes" when introducing the new concepts